.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
```

Comment:
Nice! I started writing a list with extension tips/recommendations (after you asked about this) that I'll put in Notion. Since we have a bunch of repos that _might_ be a more scalable approach? Or the two can complement each-other. Anyway, great initiative! ✨

Reply:
I see. Perhaps the two complement each other. I just thought would be more convenient having a vscode extensions under a vscode configuration.

Reply:
No sure, let's do both 😊